### DISCUSSION TITLE
there is a ssl validation fails to access s3 in python sdk without proxy

### DISCUSSION THREAD

**Discussion by jennylover** [aws/sagemaker-python-sdk#3656] [Help]
this is a sample code,

        self.est = TensorFlow(
            entry_point = 'train.py',
            source_dir='code',
            role=sagemaker.role,
            framework_version='2.3.1',
            model_dir=False,
            py_version='py37',
            instance_type=instance_type,
            instance_count=1,
            output_path=s3_output_path_uri,
            hyperparameters={
                'batch-size':512,
                'epochs':10,
                'learning-rate': 1e-3,
                'beta_1':0.9,
                'beta_2':0.999
            },
        )

        self.est.fit(inputs=channels)

and when python sdk calls TensorFlow estimator, it returns botocore.exceptions.SSLError: SSL validation failed for https://s3-location [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1129)

boto3 can work with proxy configuration, but could not find how to set proxy with sagemaker python sdk.

        self.client = boto3.client('machinelearning',
                            AWS_REGION,
                            aws_access_key_id=AWS_ACCESS_KEY_ID,
                            aws_secret_access_key=AWS_SECRET_ACCESS_KEY,
                            config=Config(proxies={'https':'XX.XX.XX.XX:8080'},signature_version='s3v4'),
                            verify='cert.crt',
                            )

any chance to set proxy configuration in sagemaker python sdk?

Thanks,

**Comment by ntw-au** [comment]
I have tried to set a custom certificate bundle several ways but so far the SageMaker Python SDK does not respect the configuration. I've raised this as a feature request: https://github.com/aws/sagemaker-python-sdk/issues/5108
